### PR TITLE
Hungarian translation and minor improvements

### PR DESCRIPTION
**Pull request by radokristof**
Added Hungarian translation.
And also added toLocalString() format to the lastUpdate information. Otherwise the string is pretty long and usually contains not so relevant or important information (like current Timezone, etc...).

### CODE REVIEW

File: MMM-COVID19.js [hunk 1]
```diff
@@ -288,7 +289,7 @@ Module.register("MMM-COVID19", {
       // convert API date/time UTC to local timezone
       let dateToLocalTimezone = new Date(this.countriesStats['statistic_taken_at'] + ' UTC')
 
-      statsDateCell.innerHTML = this.translate('statistic taken at ') + dateToLocalTimezone
+      statsDateCell.innerHTML = this.translate('statistic taken at ') + dateToLocalTimezone.toLocaleString()
```
Comment (bibaldo) [inline MMM-COVID19.js:292]: @radokristof an ongoing PR (https://github.com/bibaldo/MMM-COVID19/pull/36) is in process of approval with these kind of changes (time and date format). So please remove this from your PR

File: translations/hu.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    "Deaths": "Elhunytak",
+    "New Deaths": "Mai elhunytak",
+    "Serious" : "Súlyos esetek",
+    "Recovered": "Gyógyult",
+    "Active": "Aktív",
+    "Cases Per M" : "Esetek/millió",
+    "Worldwide" : "Világszerte",
+    "statistic taken at ": "Statisztika utoljára frissítve ",
+    "N/A" : "N/A"
+}
```
Comment (bibaldo) [inline translations/hu.json:13]: @radokristof it's all ok, but the country list hungarian translations are missing. 
Please add them in order to have the PR approved.

File: MMM-COVID19.js (hunk 1 above)
Comment (radokristof) [reply MMM-COVID19.js:292]: I just saw that now. I will remove this.

File: translations/hu.json (hunk 2 above)
Comment (radokristof) [reply translations/hu.json:13]: Where are this translations are?

**Comment by radokristof** [comment]
All countries have been added and translated

**Pull request merged**